src/components/entity/ha-entity-toggle.js

```diff
@@ -21,11 +21,10 @@ class HaEntityToggle extends PolymerElement {
       paper-icon-button[state-active] {
         color: var(--paper-icon-button-active-color, var(--primary-color));
       }
-      paper-toggle-button {
-        cursor: pointer;
-        --paper-toggle-button-label-spacing: 0;
-        padding: 13px 5px;
-        margin: -4px -5px;
+      mwc-switch {
+        display: inline-block;
+        padding: 17px 5px 13px;
```

Comment:
`ha-entity-toggle` renders a toggle or 2 icon buttons in case it's an assumed state. We want to have the padding on the switch directly as it improves the hit target. The circle of the switch is what overlays the bounding box it seems.

Reply:
If the state badge had a picture, it would also touch the edges btw.

Reply:
I was more looking at the fact that it is not vertically centered - it's too low. It's much closer to center in the state-rows. It's pretty minor and I think it's fine for now, and we can just fix the positioning in the LL toggle-row later

Reply:
Ah the ripple, didn't notice that. Yeah something is off with the margin and padding in ha-entity-toggle